### ISSUE TITLE
what's detail info of your v5

### ISSUE THREAD

**Issue by zwj0571** [V5OSX/clover#1]
can you tell me your hardware info, is it i7 model?

**Comment by notly** [comment]
@zwj0571 https://github.com/Kaijun/Acer-V5-573G-Hackintosh

mine is i5 1080p acer v5-573g, 
**you can find the right dsdt and replace them in  CLOVER/ACPI/patched/DSDT.aml**

**Comment by zwj0571** [comment]
@llsf have any diff with your repo？ can i use your clover to boot ei 10.11.5

**Comment by zwj0571** [comment]
I have install 10.11 by your clover and kext. I have 573g with i5 4200U 1080, which I think is the same model with yours. but there still have some issue i need to solve
1. audio, there is no audio device found in my EI. dummy audio.kext was installed
2.the graphic lag when boot. I need to logout..

**Comment by notly** [comment]
ok, sry for that, i will share the Dummy.kext as soon as possible

the graphic lag  maybe caused by the wrong dsdt, can you offer your full details of you computer?
xd

**Comment by zwj0571** [comment]
thank you for your reply. I found in your kexts there is a Dummy.kext. Is that means i need to install it in SLE fold, not only put in clover/kexts?
basicly, my cpu is i5 4200 with 750m

**Comment by notly** [comment]
just same as my pc!!!

donot touch the original applehda.kext

install dummy.kext will get the audio work!
graphic lag : what version of bios you are using ? bios 2.29 and 2.30 is ok ! for now!

**Comment by zwj0571** [comment]
yeah, i use bios2.30.there is some weird problem in this bois . i try boot EI by using a usb driver in UEFI mode . so i have to add cloverx64.efi in to trust file in bios. but.when i try to delete it. it may fail,  after that,i can nerver boot ei .i have to refresh bios by changing platform.ini in bios flash tool to clear this set.

**Comment by zwj0571** [comment]
@llsf the audio does not work even I install your dummy.kext to SLE or LE. I version of EI is 10.11.5, I think the dummy.kext in your repo is too old? I have create a repo, in my repo i use applealc.kext to solve the problem
